structures/MisakiClient.js

```diff
@@ -68,17 +68,15 @@ class MisakiClient extends Client {
     try {
       const command = new (require(`${commandPath}${path.sep}${commandName}`))(this);
       command.location = commandPath;
-      if (command.init) await command.init();
       this.commands.set(command);
-    } catch (e) {
-      this.logger.error(`Unable to load command ${commandName}: ${e}`);
+    } catch (err) {
+      this.logger.error(`Unable to load command ${commandName}: ${err}`);
     }
   }
 
   async unloadCommand(commandPath, commandName) {
     const command = this.commands.get(commandName);
     if (!command) return `The command \`${commandName}\` doesn"t seem to exist, nor is it an alias. Try again!`;
-    if (command.shutdown) await command.shutdown();
```

Comment:
Why was this removed? Isn't it needed for certain commands?

Reply:
No otherwise it would've been kept

Reply:
iirc, that was there if commands had an init function, for example York-Dev had init functions for the tags and examples

Reply:
Still not used in Misaki...